src/components/ShoppingBag/types.ts

```diff
@@ -0,0 +1,7 @@
+export type ShoppingListItem = {
+  name: string;
+  quantity: number;
```

Comment:
we have to think about quantity type in the future, as for now its a string because it could be something like "3 tablespoons", so we need to be aware of that, maybe separate it into number and unit or something like that. for now its all good

Reply:
Quantity here is used for only the shopping list context, i.e. bags of sugar, cartons of milk, bottles of sauce, etc.